pyscriptjs/tests/integration/test_splashscreen.py

```diff
@@ -1,7 +1,12 @@
+import pytest
 from playwright.sync_api import expect
 
 from .support import PyScriptTest, skip_worker
 
+pytest.skip(
+    reason="DECIDE: Should we remove the splashscreen?", allow_module_level=True
```

Comment:
yes!

Reply:
+1000 from me but iirc someone had concerns so I'd prefer to hear their pitch first

Reply:
the community has ben loud about this though ... and it takes no more than a second now to bootstrap pyodide + with workers it has no reason to exist ... I say get this intrusive solution of the project and let's make it a plugin **opt-in** for whoever misses it once we're out with next.

Reply:
Kill the splash screen for all the reasons @WebReflection states. ☠️

If folks want a splash screen, they're likely technical enough to know how to implement one themselves.